deliver/lib/assets/summary.html.erb

```diff
@@ -174,12 +174,23 @@
             <% sc = @screenshots.find_all { |s| s.language == language } %>
             <% sc_by_size = sc.group_by { |i| i.screen_size } %>
 
+            <% trailers_sorted = @trailers.find_all { |t| t.language == language } %>
+
             <% sc_by_size.keys.sort.each do |screen_size| %>
               <% screenshots = sc_by_size[screen_size].sort { |a, b| [a.path] <=> [b.path] } %>
+              <% #Let's see if we have trailer for that device %>
+              <% trailer = trailers_sorted.find { |t| t.device_type == screenshots[0].device_type } %>
               <%# we are guaranteed to have at least one element because of the group_by %>
               <h4><%= screenshots[0].formatted_name %></h4>
               <div class="app-screenshot-row">
 
+              <% if trailer %>
+                <video width="400px" height="300px" controls poster="<%= trailer.preview_image_path %>">
```

Comment:
move width/height to CSS and use % instead (see existing `app-screenshot` class one)